pkg/handler/eventhandler_suite_test.go

```diff
@@ -30,6 +31,18 @@ func TestEventhandler(t *testing.T) {
 	RunSpecsWithDefaultAndCustomReporters(t, "Eventhandler Suite", []Reporter{envtest.NewlineReporter{}})
 }
 
+var testenv *envtest.Environment
+var cfg *rest.Config
+
 var _ = BeforeSuite(func() {
 	logf.SetLogger(logf.ZapLoggerTo(GinkgoWriter, true))
+
+	testenv = &envtest.Environment{}
+	var err error
+	cfg, err = testenv.Start()
+	Expect(err).NotTo(HaveOccurred())
```

Comment:
can we just extract this out onto a helper on `testenv` at some point?  I want to be able to call `BeforeSuite(testenv.Environment{}.BeforeSuite)` or something similar.

Reply:
Should I create an issue to track this or is it already tracked some other place?